internal/cmd/root.go

```diff
@@ -16,6 +16,10 @@ var (
 )
 
 func Root() *cobra.Command {
+	exitAfterRun := false
```

Comment:
Having `tuiCmd` flags here while they belong to `tuiCmd` as well as interleaving `tuiCmd` with the root command might be improved a bit.

In `root.go`, I would add these lines:

```go
tuiCmd := tuiCmd()
// tuiCmd is also a default command
cmd.RunE = tuiCmd.RunE
// remember to copy flags
tuiCmd.Flags().VisitAll(func(f *pflag.Flag) {
	pflags.AddFlag(f)
})
```

And:

```go
cmd.AddCommand(tuiCmd)
```

In this way, we keep `tuiCmd` totally unaware that it is used in any different way than other commands.

Reply:
The flag copying code errors because it re-defines the flags set by `setDefaultFlags`. In the most recent PR I used the following code:

```go
	tuiCmd.Flags().VisitAll(func(f *pflag.Flag) {
		if flag := cmd.Flags().Lookup(f.Name); flag == nil {
			cmd.Flags().AddFlag(f)
		}
	})
```

The alternative would be to never run `setDefaultFlags` on the root command and let them cascade from the tui command. I felt that it was better to be explicit than implicit here, even if at the expense of verbosity. Let me know if you want me to do it the other way!